static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -326,3 +326,32 @@ export function getDiseaseCommonName(data: GraphNodes): string {
     return formattedDiseaseName;
   }
 }
+/**
+ * Checks whether the stat var for medical condition of disease exists
+ * @param data
+ * @returns boolean indicating if desired stat var exists
+ */
+export function doesDiseasePrevalenceIDexist(data: GraphNodes): boolean {
+  // sets the default value of the boolean as false
+  if (!data) {
+    return false;
+  }
+  // check for null values
+  if (_.isEmpty(data.nodes) || _.isEmpty(data.nodes[0].neighbors)) {
```

Comment:
nit: can combine this check with the check on line 336?

Reply:
Done!